tests/Common/ThrowingTraceListener.cs

```diff
@@ -1,7 +1,9 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
 
+#pragma warning disable IDE0005
```

Comment:
Remove this change from this PR. If this is causing error then it should be changed in main separately. Pipeline migration shouldn't cause this.